tests/unit/core/support_modules/test_msg_buf.cpp

```diff
@@ -19,8 +19,8 @@
 
 // Test the rdmnet/private/msg_buf.c module
 // The msg_buf module is the TCP stream parser that parses the RDMnet TCP-based protocols: Broker,
-// RPT and EPT. This module works using a small testing library which deserializes a set of golden
-// master RDMnet protocol messages which live in tests/data/messages and validates them.
+// RPT and EPT. This module works using a small testing library which deserializes a set of RDMnet
```

Comment:
"Golden master" is meaningful here, and I don't believe it uses the term "master" in an offensive context. This change should be reverted.